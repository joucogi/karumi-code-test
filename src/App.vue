<template>
  <div id="app">
    <router-view />
  </div>
</template>

<style>
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  padding: 50px;
  margin: 0 auto;
  display: flex;
  flex-direction: row;
  align-items: stretch;
  height: 100%;
}

#app > div {
  margin: 0 auto;
}
</style>
