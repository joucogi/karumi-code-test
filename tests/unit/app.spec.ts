import { shallowMount, Wrapper } from "@vue/test-utils";
import Vue from "vue";
import VueRouter from "vue-router";

import App from "@/App.vue";
import { toMatchSnapshot } from "./helpers";

Vue.use(VueRouter);

let wrapper: Wrapper<Vue>;

beforeEach(() => {
  wrapper = shallowMount(App);
});

describe("App.vue", () => {
  it("renders correctly", () => {
    toMatchSnapshot(wrapper);
  });
});
