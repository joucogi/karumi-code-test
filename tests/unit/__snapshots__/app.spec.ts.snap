// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders correctly 1`] = `
<div
  id="app"
>
  <router-view-stub
    name="default"
  />
</div>
`;
